src/js/components/DescriptionList.js

```diff
@@ -23,11 +23,14 @@ class DescriptionList extends React.Component {
 
       // Check whether we are trying to render an object that is not a
       // React component
-      if (typeof value === 'object' && !Array.isArray(value) &&
-        value !== null && !React.isValidElement(value)) {
+      if (typeof value === 'object' && value !== null &&
+        !React.isValidElement(value)) {
 
+        // TODO (orlandohohmeier): indicate (e.g. indent) sub list for better ux
```

Comment:
I'd suggest to json.stringify objects into a `<pre>` like in Marathon UI. No need for sublists.